apps/docs/.vitepress/components/PageContributors.vue

```
@@ -0,0 +1,59 @@
+<script setup lang="ts">
+import { useData } from 'vitepress';
+import { computed } from 'vue';
+import _contributors from 'element-plus-x-metadata/dist/component-contributors.json';
```

ESLint

[error] Expected "element-plus-x-metadata/dist/component-contributors.json" to come before "vue".

(perfectionist/sort-imports)